piton/verif/diag/assembly/include/riscv/ariane/crt.S

```diff
@@ -162,6 +162,18 @@ _start:
   sll sp, sp, STKSHIFT
   add sp, sp, tp
 
+  li t1, 0
```

Comment:
I'm inclined to say we should put this under an ifdef. Maybe `PITON_NOC_IPI` ? Then we could add `-DPITON_NOC_IPI` to `gcc_args` in `sims,2.0` when building for blackparrot